### PR TITLE
start testing min required dependencies

### PR DESCRIPTION
**Pull request by mathause**
- [ ] Towards #175 (but not closing it)
 - [ ] Tests added
 - [ ] Passes `isort . && black . && flake8`
 - [ ] Fully documented, including `CHANGELOG.rst`

Start testing the lowest version of our supported dependencies. We don't specify the all of them as of yet. But with #396 and #397 this becomes relevant.

### CODE REVIEW

**Comment by mathause** [comment]
Actually version 0.9 of regionmask was required even before (for the new code path)...

**Comment by mathause** [comment]
Gentle ping @veni-vidi-vici-dormivi

**Review by veni-vidi-vici-dormivi — APPROVED** [review]

File: environment.yml
[the remark this answers is not in the document]
Comment (mathause) [reply environment.yml:1]: > sphinx

sphinx is only required for the docs and it's included in `docs/environment.yml`

> scipy

Good catch & yes that is correct; we should absolutely clean that up; see also #364.

You can find the who requires scipy using: 
```
conda activate mesmer_dev
mamba repoquery whoneeds scipy
```

File: .github/workflows/ci-additional.yml
[the remark this answers is not in the document]
Comment (mathause) [reply .github/workflows/ci-additional.yml:1]: historical reasons probably... I'll merge the two

File: environment.yml
[the remark this answers is not in the document]
Comment (veni-vidi-vici-dormivi) [reply environment.yml:1]: Aha I get it, thanks for the clarification!

**Pull request merged**